sap/rest/gcts/simple.py

```diff
@@ -30,9 +31,25 @@ def fetch_repos(connection):
     return [Repository(connection, repo['name'], data=repo) for repo in result]
 
 
+def _wait_for_clone(repo, wait_for_ready, http_exc):
+    """Wait for clone process to finish"""
+
+    start_time = time.time()
+    while time.time() - start_time < wait_for_ready:
+        repo.wipe_data()
+        try:
+            if repo.status == 'READY':
+                return
+
+        except HTTPRequestError:
+            _mod_log().debug('Failed to get status of the repository %s', repo.name)
+
```

Comment:
We shall users know that waiting was not successfull and the timeout period was not long enough or there was another serious problem. Coud we get some details from Activities? What happens when we try to import a repo which contains syntax errors?